set-theory.html.markdown

```diff
@@ -3,136 +3,123 @@ category: Algorithms & Data Structures
 name: Set theory
 contributors:
 ---
-The set theory is a study for sets, their operations, and their properties. It is the basis of the whole mathematical system.
+Set theory is a branch of mathematics that studies sets, their operations, and their properties.
 
-* A set is a collection of definite distinct items.
+* A set is a collection of disjoint items.
 
-## Basic operators
-These operators don't require a lot of text to describe.
+## Basic symbols
 
-* `∨` means or.
-* `∧` means and.
-* `,` separates the filters that determine the items in the set.
+### Operators
+* the union operator, `∪`, pronounced "cup", means "or";
+* the intersection operator, `∩`, pronounced "cap", means "and";
+* the exclusion operator, `\`, means "without";
+* the compliment operator, `'`, means "the inverse of";
+* the cross operator, `×`, means "the Cartesian product of".
 
-## A brief history of the set theory
-### Naive set theory
-* Cantor invented the naive set theory.
-* It has lots of paradoxes and initiated the third mathematical crisis.
+### Qualifiers 
+* the colon qualifier, `:`, means "such that";
+* the membership qualifier, `∈`, means "belongs to";
+* the subset qualifier, `⊆`, means "is a subset of";
+* the proper subset qualifier, `⊂`, means "is a subset of but is not equal to".
 
-### Axiomatic set theory
-* It uses axioms to define the set theory.
-* It prevents paradoxes from happening.
+### Canonical sets
+* `∅`, the empty set, i.e. the set containing no items;
+* `ℕ`, the set of all natural numbers;
+* `ℤ`, the set of all integers;
+* `ℚ`, the set of all rational numbers;
+* `ℝ`, the set of all real numbers.
 
-## Built-in sets
-* `∅`, the set of no items.
-* `N`, the set of all natural numbers. `{0,1,2,3,…}`
```

Comment:
I find these examples for natural numbers, integers etc helpful, I think they shouldn't be deleted